lib/tasks/teem_tasks.rake

```diff
@@ -0,0 +1,111 @@
+require 'rake'
+
+ORGANIZATION_ID = 13856
+
+# Removes group members from Teem that are no longer in Roles Management.
+#
+# +token+ is a string containing the access token used to authorize against Teem API
+# +teem_users_to_remove+ is an array of strings containing users in Teem that need to be removed
+# +all_user_ids+ is an array of strings containing a mapping from user email to id in Teem
+# +group_id+ is an integer containing the group id
+def remove_teem_group_members(token, teem_users_to_remove, all_user_ids, group_id)
```

Comment:
Rearrange the parameters in order of context: token, group in question, members to remove, e.g. `def remove_teem_group_members(token, group_id, users_to_remove)`. Also, is there a good way to refactor this such that `all_user_ids` isn't needed? It seems odd to say "here's the list of users to remove, and also here's this thing you'll need to decode it", right?

Reply:
Would it be better if instead of passing in `all_user_ids` as a parameter I called its function `Teem.user_ids()` instead every time to get the mapping or should I get rid of this concept altogether. I used this mapping because a user can not be deleted via email, thus, I used `all_user_ids` to get the accompanying id for that to take place.